reana_server/scheduler.py

```diff
@@ -141,6 +144,8 @@ def requeue_workflow(self, **kwargs):
                 kwargs["parameters"],
                 priority=kwargs.get("priority", 0),
                 min_job_memory=kwargs.get("min_job_memory", 0),
+                retry_count=kwargs.get("retry_count", 0) + 1,
+                delay=REANA_SCHEDULER_RETRY_DELAY,
             )
             logging.error(
                 f"Requeueing workflow " f'{kwargs["workflow_id_or_name"]} ...'
```

Comment:
BTW as part of this PR we could perhaps clean the logs.  Currently, when REANA cluster is not ready, I'm seeing:

```
2021-06-10 14:19:15,385 | root | MainThread | INFO | REANA not ready to run workflow hello.3. Requeueing workflow and retrying in 1 second(s) ...
2021-06-10 14:19:15,399 | root | MainThread | ERROR | Requeueing workflow hello.3 ...
2021-06-10 14:19:20,559 | root | MainThread | INFO | REANA not ready to run workflow hello.2. Requeueing workflow and retrying in 1 second(s) ...
2021-06-10 14:19:20,579 | root | MainThread | ERROR | Requeueing workflow hello.2 ...
2021-06-10 14:20:15,409 | root | MainThread | INFO | REANA not ready to run workflow hello.3. Requeueing workflow and retrying in 1 second(s) ...
2021-06-10 14:20:15,424 | root | MainThread | ERROR | Requeueing workflow hello.3 ...
2021-06-10 14:20:20,590 | root | MainThread | INFO | REANA not ready to run workflow hello.2. Requeueing workflow and retrying in 1 second(s) ...
2021-06-10 14:20:20,604 | root | MainThread | ERROR | Requeueing workflow hello.2 ...
2021-06-10 14:21:15,433 | root | MainThread | INFO | REANA not ready to run workflow hello.3. Requeueing workflow and retrying in 1 second(s) ...
2021-06-10 14:21:15,447 | root | MainThread | ERROR | Requeueing workflow hello.3 ...
2021-06-10 14:21:20,615 | root | MainThread | INFO | REANA not ready to run workflow hello.2. Requeueing workflow and retrying in 1 second(s) ...
2021-06-10 14:21:20,630 | root | MainThread | ERROR | Requeueing workflow hello.2 ...
2021-06-10 14:22:15,456 | root | MainThread | INFO | REANA not ready to run workflow hello.3. Requeueing workflow and retrying in 1 second(s) ...
2021-06-10 14:22:15,470 | root | MainThread | ERROR | Requeueing workflow hello.3 ...
```

The ERROR level seems to say that there was something wrong, but actually it is only the requeueing that is happening... So something like INFO displayed above is already enough.  Two options: (1) delete the ERROR message; (2) turn it into INFO or WARNING rather than ERROR.  Let's keep ERROR only for situations where our code base may malfunction...

Reply:
Otherwise the retry count works well for me :+1:

Reply:
Changed the logging level to `INFO`